masterlist.yaml

```diff
@@ -21822,53 +21822,33 @@ plugins:
         itm: 6
         udr: 4
 
-  - name: 'HESU .*\.esp'
+  ### The Hesu Mod Collection ###
+  - name: 'HESU (AppleHotSprings|DiveRock|FairmereVillage|GnollMountain|GreatForestGorge|HellLands|KoyoTown|PoisonSwamp|SeasideSettlement|Shipton|SkyrimTemple|SmokeTown|ValenwoodMine|WhiteRoseRiver|WolfsBorough)\.esp'
     url:
       - link: 'https://www.afkmods.com/index.php?/files/file/2344-the-hesu-mod-collection/'
         name: 'The Hesu Mod Collection'
     req: [ 'DLCShiveringIsles.esp' ]
+    msg:
+      - <<: *compatPatch
+        subs: [ '[The Hesu Mod Collection - Hesu Mods Patches](https://www.afkmods.com/index.php?/files/file/2344-the-hesu-mod-collection/)' ]
+        condition: 'not file(".*((AppleHotSprings|DiveRock|Fairmere|GnollMountain|GreatForestGorge|HellLands|KoyoTown|PoisonSwamp|Seaside|Shipton|SkyrimTemple|SmokeTown|ValenwoodMine|WolfsBorough|WRR).*( Patch)|MTCExpanded\+HesuPoisonSwamp|TOS\+Hesu Poison Swamp|ValenwoodMine\+DaedricShrineRuins)\.esp")'
```

Comment:
Are these `( Patch)`s supposed to be `( Patch)?`s? The capture groups are redundant otherwise.